examples/modules/cfd/pages/spring/README.adoc

```diff
@@ -36,15 +36,17 @@ The case data files are available in Github link:{uri-data}/spring/[here]:
 
 == Geometry
 
-image:spring_cfd/springmesh.png[50%]
+The geometry consists of a spring shaped tube with one inlet and one outlet.
+
+image:spring/mesh.png[50%]
 
 
 == Input parameters
 
 [options="header"]
 |===
 | Name |Description | Value | Unit |
-| stem:[\mu_f] | fluid velocity | 6e-3  | stem:[Pa.s] |
+| stem:[\mu_f] | fluid viscosity | 6e-3  | stem:[Pa.s] |
 | stem:[\rho] | fluid density | 1056 | stem:[kg/m^3] |
 |===
```

Comment:
discuss the reynolds number based on these data, the geometry data as well as the inflow velocity. Moreover you have set boundary conditions, they need to be presented/discussed.